src/cgame/cg_local.h

```diff
@@ -1538,8 +1535,8 @@ typedef struct
 	sfxHandle_t fallSound;
 	sfxHandle_t turretSpinupSound;
 
-	sfxHandle_t hardBounceSound1;
-	sfxHandle_t hardBounceSound2;
+	sfxHandle_t grenadeBounceSound0;
```

Comment:
Why zero index names? I think zero indexing only makes sense for computers, not humans.

Reply:
well, why not. \^_\^ can change it to start with 1 again if needed.

Reply:
we also have a level0 model by the way, this is a bit inconsistent in fact

Reply:
I guess it was because of that:

```
$ find . -name '*0.*'
./models/weapons/abuildupg/flash0.flac
./models/weapons/abuildupg/impactflesh0.flac
./models/weapons/blaster/flash0.flac
./models/weapons/firebomb/impact0.flac
./models/weapons/grenade/impact0.flac
./models/weapons/grenade/bounce0.flac
./models/weapons/lcannon/flash0.flac
./models/weapons/lcannon/impact0.flac
./models/weapons/level0/flash0.flac
./models/weapons/level2/flash0.flac
./models/weapons/level2/impactflesh0.flac
./models/weapons/level3/flash0.flac
./models/weapons/level3/impactflesh0.flac
./models/weapons/level3upg/flash0.flac
./models/weapons/level4/flash0.flac
./models/weapons/level4/impactflesh0.flac
./models/weapons/lgun/flash0.flac
./models/weapons/mdriver/flash0.flac
./models/weapons/prifle/flash0.flac
./models/weapons/prifle/impact0.flac
./models/weapons/rifle/flash0.flac
./models/weapons/rifle/ricochet0.flac
./models/weapons/shotgun/flash0.flac
./models/weapons/lockblob/impact0.flac
./sound/buildables/human/damage0.flac
```

Reply:
I guess it's fine, just odd to me :-/ I'm not sure relying on existing stuff is a good argument given that the whole reason we are doing this is to change existing paradigms into more reasonable ones :)

Reply:
sure, but this one does not looks bad :-)